cob_twist_controller/cfg/TwistController.cfg

```diff
@@ -7,25 +7,32 @@ gen = ParameterGenerator()
 
 damping_method_enum = gen.enum([ 
                        gen.const("manipulability",      int_t, 0, "manipulability"),
-                       gen.const("constant",            int_t, 1, "constant"),
-                       gen.const("truncation",          int_t, 2, "truncation")],
+                       gen.const("constant",            int_t, 1, "constant"),], 
                      "enum types for the damping_method")
 
-gen.add("damping_method", int_t, 0, "The damping method to use (edited via an enum)", 0, 0, 2, edit_method=damping_method_enum)
+constraints_enum = gen.enum([ 
```

Comment:
Can we call this wln_constraint so the meaning is clear in the AugmentedSolverParams data structure?
(Having future extensions in mind...)